plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InfoCommand.java

```diff
@@ -62,7 +63,7 @@ public void execute(FunnyGuilds plugin, PluginConfiguration config, MessageConfi
             messageLine = StringUtils.replace(messageLine, "{REGION-SIZE}", config.regionsEnabled ? String.valueOf(guild.getRegion().getSize()) : messages.gRegionSizeNoValue);
             messageLine = StringUtils.replace(messageLine, "{GUILD-PROTECTION}", protectionEndTime < now ? "Brak" : TimeUtils.getDurationBreakdown(protectionEndTime - now));
 
-            Rank rank = guild.getRank();
+            GuildRank rank = guild.getRank();
             messageLine = StringUtils.replace(messageLine, "{POINTS-FORMAT}", IntegerRange.inRangeToString(rank.getPoints(), config.pointsFormat));
             messageLine = StringUtils.replace(messageLine, "{POINTS}", Integer.toString(rank.getPoints()));
```

Comment:
Ogólnie nie widzę sensu w robieniu sumy i średniej punktów gildii, bo ktoś może się z tym pomylić.
I tutaj jest chyba błąd właśnie przez wprowadzenie tej sumy.

Proponuję zrobić getPoints jako średnia i bez tej sumy, która chyba nigdzie nie jest używana.

Reply:
Jest to związane z tym, że realizując GH-736 chce przy okazji uwolnić po czym gracz chce mieć ranking. Żeby to nie było sztywno ustawione na punkty.